app/src/main/java/com/netmera/netmerafintech/data/Enums.kt

```diff
@@ -0,0 +1,7 @@
+package com.netmera.netmerafintech.data
+
+enum class CardType(val value: Int) {
+    CARD_BLACK(0),
```

Comment:
Don't need to add card anymore. You can simply name it as `Black`, `Blue` and `Yellow`